data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/RecordDispatcherImpl.java

```diff
@@ -126,7 +128,7 @@ public RecordDispatcherImpl(
    * @param record record to handle.
    */
   @Override
-  public Future<Void> dispatch(KafkaConsumerRecord<Object, CloudEvent> record) {
+  public Future<Void> dispatch(KafkaConsumerRecord<Object, CloudEvent> record, ConsumerRecordContext recordContext) {
```

Comment:
since ConsumerRecordContext contains the record can we pass only `ConsumerRecordContext` here.